ompi/mca/coll/adapt/coll_adapt_component.c

```diff
@@ -53,19 +53,19 @@ mca_coll_adapt_component_t mca_coll_adapt_component = {
 	    OMPI_RELEASE_VERSION,
 	    
 	    /* Component functions */
-	    adapt_open,               /* open */
-	    adapt_close,
-	    NULL,                     /* query */
-	    adapt_register
+	    ompi_coll_adapt_open,       /* open */
+	    ompi_coll_adapt_close,
+	    NULL,                       /* query */
+	    ompi_coll_adapt_register
```

Comment:
```suggestion
	    .mca_open_component = ompi_coll_adapt_open,
	    .mca_close_component = ompi_coll_adapt_close,
	    .mca_register_component_params = ompi_coll_adapt_register
```
whith this, you keep in mind what you are setting and you do not need NULL lines

Reply:
you can apply this to l.50-53
see [coll_tuned_component.c](https://github.com/Bullrp4u/ompi/blob/758c3f8d05a476248db979a45fc16c961986285c/ompi/mca/coll/tuned/coll_tuned_component.c#L97-L109)
`coll_tuned_component.c

            .collm_version = {
            MCA_COLL_BASE_VERSION_2_0_0,

            /* Component name and version */
            .mca_component_name = "tuned",
            MCA_BASE_MAKE_VERSION(component, OMPI_MAJOR_VERSION, OMPI_MINOR_VERSION,
                                  OMPI_RELEASE_VERSION),


            /* Component open and close functions */
            .mca_open_component = tuned_open,
            .mca_close_component = tuned_close,
            .mca_register_component_params = tuned_register,
        },
`

Reply:
Done in 8b854f9fe3a23e39e3d3511808406b43bdf5c7d2